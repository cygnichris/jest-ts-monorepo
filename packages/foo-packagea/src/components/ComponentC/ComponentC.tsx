import * as React from 'react';
import { ComponentB } from '@foo/packageb'
export const ComponentC = () => <div ><ComponentB /></div>
