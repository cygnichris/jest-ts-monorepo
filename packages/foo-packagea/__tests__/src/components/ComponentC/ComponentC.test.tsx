import * as React from 'react';
import { ComponentC } from '@asrc/components/ComponentC/ComponentC';
import { render } from '@testing-library/react';

describe('<ComponentC/> Snapshot', () => {
  it('renders', () => {
    const { asFragment } = render(
      <ComponentC />
    );

    expect(asFragment()).toMatchSnapshot();
  })
})


