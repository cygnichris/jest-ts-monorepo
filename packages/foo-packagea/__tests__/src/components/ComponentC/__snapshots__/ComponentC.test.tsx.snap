// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ComponentC/> Snapshot renders 1`] = `
<DocumentFragment>
  <div>
    <div>
      Hello World 
    </div>
  </div>
</DocumentFragment>
`;
